source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -369,6 +369,25 @@ progress.
 For more information on DDL operations, see
 :ref:`txn-prod-considerations-ddl`. 
 
+Network Latency
+~~~~~~~~~~~~~~~
+
+Network latency or long physical distances between migration components
+can negatively affect sync speed.
+
+Latency between ``mongosync`` and destination shards
+  For each change event that ``mongosync`` applies, ``mongosync`` does 
+  an extra roundtrip to its metadata stored on the destination.
+
+Latency between destination shards
+  For each change event that ``mongosync`` applies, it must access a 
+  document that may live on a different shard. This can result in many
+  cross-shard transactions.
```

Comment:
```suggestion
  Mongosync writes operations and updates its own metadata on the destination cluster in batches in a transaction. This can result in cross-shard transactions, which may be more costly if the shards are far apart.
```